corehq/motech/openmrs/tasks.py

```diff
@@ -46,7 +46,10 @@
 from corehq.motech.utils import b64_aes_decrypt
 
 RowAndCase = namedtuple('RowAndCase', ['row', 'case'])
-LOCATION_OPENMRS = 'openmrs_uuid'  # The location metadata key that maps to its corresponding OpenMRS location UUID
+# REQUEST_TIMEOUT is 5 minutes, but reports can take up to an hour
+REPORT_REQUEST_TIMEOUT = 60 * 60
```

Comment:
yikes - that's crazy! Is there no other way to do this?

Reply:
There are 2 other options to explore, but we can't use them immediately:
1. [The Reporting API documentation](https://github.com/openmrs/openmrs-module-reportingrest/wiki) mentions a "reportRequest" API endpoint, which "schedules a report to be evaluated (asynchronously)" ... but doesn't give any more details. Trying that endpoint with and without a report UUID (the way the "reportdata" endpoint works) both result in errors and Java stacktraces. But this sounds like it has potential, and I'll follow up by reading [the code](https://github.com/openmrs/openmrs-module-reportingrest), and asking on [the forum](https://talk.openmrs.org).
2. The other option is to use [asynchronous requests](https://aiohttp.readthedocs.io/en/stable/client_quickstart.html) so we don't have to block waiting for slow API responses.